appinventor/components/src/com/google/appinventor/components/runtime/PasswordTextBox.java

```diff
@@ -54,4 +58,20 @@ public PasswordTextBox(ComponentContainer container) {
     view.setImeOptions(EditorInfo.IME_ACTION_DONE);
 
   }
+
+  private boolean passwordVisible;
```

Comment:
Java convention is to define fields at the top of the class definition. You should move this definition of passwordVisible to before the constructor.